provisioning/datastore.yml

```diff
@@ -0,0 +1,84 @@
+- name: use custom .bashrc
+  copy: src=bashrc-datastore dest=~/.bashrc backup=yes
+
+- name: ensure /srv directory exists
+  file: path=/srv state=directory
+  become: true
+
+- name: ensure /srv/datastore directory exists
+  file: path=/srv/datastore state=directory
+        owner=hisparc group=hisparc mode=2775
+  become: true
+
+- name: set default group write ACLs on /srv/datastore
+  command: /usr/bin/setfacl -m d:g::rwx /srv/datastore
+  become: true
+
+- name: ensure /var/log/hisparc directory exists
+  file: path=/var/log/hisparc state=directory group=hisparc mode=775
+  become: true
+
+- name: checkout datastore in /srv/datastore/code
+  git: repo=https://github.com/HiSPARC/datastore.git dest=/srv/datastore/code
+       update=yes force=no
+  notify: restart uWSGI
+
+- name: remove compiled python code
+  command: find /srv/datastore/code -name *.pyc -delete
+
+- name: Copy conda requirements files
+  copy: src=conda-datastore.list dest=/home/{{ ansible_user_id }}/
+
+- name: Copy pip requirements files
+  copy: src=pip-datastore.list dest=/home/{{ ansible_user_id }}/
+
+- name: create conda environment
+  shell: creates=/srv/datastore/datastore_env
+         yes | /opt/miniconda/bin/conda create
+         -p /srv/datastore/datastore_env python --yes
```

Comment:
I would like to suggest:
 - add python=2.7 to `conda-datastore.list`
 - add `--file /home/{{ ansible_user_id }}/conda-datastore.list` to the `conda env create` command. 
 - remove the `conda install`  command

This will help us to quickly spot which version of python is running on a server. And this will help we transition the datastore to python 3.6 but leave publicdb on python 2.7.

(I did this and changed it to `python=3.6` for my personal VM)